.github/workflows/remote-copy.yml

```diff
@@ -184,31 +212,42 @@ jobs:
         id: manifest
         env:
           REMOTE_MANIFEST_FILE_LIST_PATH: ${{ vars.REMOTE_TMP_DIR }}/remote-copy-manifests-${{ github.run_id }}.log
+          LOCAL_MANIFEST_FILE_LIST_PATH: ./manifests-copied.log
+          MANIFEST_FILE_NAME: manifest.yaml
         # Generate manifests for files copied over in the earlier rsync job
         run: |
           ssh ${{ secrets.SSH_USER }}@${{ secrets.SSH_HOST_DATA }} -i ${{ steps.ssh.outputs.private-key-path }} /bin/bash <<'EOT'
           module use ${{ vars.YAMF_MODULE_PATH }}
           module load ${{ vars.YAMF_MODULE_NAME }}
 
           manifest_paths=()
-          while read -r path:
-            cd $(dirname $path)
-            manifest_dir=$(basename $path)
-            yamf add -n manifest.yaml --force $manifest_dir
-            manifest_paths+=("$manifest_dir/manifest.yaml")
-          done <<< "${{ steps.copy.outputs.paths }}"
+          for path in ${{ steps.copy.outputs.paths }}; do
+            echo "Path is $path"
+            manifest_dir=$(dirname $path)
+            cd $manifest_dir || exit
+            manifest_file=$(basename $path)
```

Comment:
I find the variable name 'manifest_file' confusing. I thought it meant the name of the manifest file. It should be sufficient to either mutate `path` at this point, as the full pathname isn't used again, or just use `filename` or similar.

Reply:
Update the variable name to `manifest_entry_filename` to note that it is the filename of an entry in the manifest file. Done in https://github.com/ACCESS-NRI/model-config-inputs/pull/3/commits/5fad05b84ed9b415f9fd649be11145291a460d22

Reply:
Thanks.